src/app.ts

```diff
@@ -57,6 +56,9 @@ app.use((req, res, next) => {
 
 // error handler
 app.use((err, req, res, next) => {
+
+  logger.error("An error has occured. Re-routing to the error screen - " + err.stack);
```

Comment:
In trying to get this working, I was getting redirected to the error page but nothing helpful was appearing in the logs. This new line ensures that the stack trace is written there when an error causes the redirection.